android/UtilsPlugin.java

```diff
@@ -268,12 +268,35 @@ public void openAppSettings() {
 
 	public void showEnableNotificationPopup(String params) {
 		try {
+			String title = "", message = "", open_settings_btn_title = "";
+			JSONObject ogData = new JSONObject(params);
+			Iterator<?> keys = ogData.keys();
+			while( keys.hasNext() ){
+				String key = (String)keys.next();
+				Object o = ogData.get(key);
+				if(key.equals("title")){
+					title = (String) o;
+					continue;
+				}
+				if(key.equals("message")){
+					message = (String) o;
+					continue;
+				}
+				if(key.equals("open_settings_btn_title")){
+					open_settings_btn_title = (String) o;
+					continue;
+				}
+			}
+
+			final String ftitle = title;
+			final String fmessage = message;
+			final String fopen_settings_btn_title = open_settings_btn_title;
```

Comment:
If final variable is needed, No while loop needed.
String title = data.optString("title", "Title");
similarly for all. So we dont need to use extra 3 variable and can avoid while loop here